plugins/SolidView/SolidView.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2015 Ultimaker B.V.
+# Uranium is released under the terms of the AGPLv3 or higher.
+
+from UM.View.View import View
+from UM.Scene.Iterator.DepthFirstIterator import DepthFirstIterator
+from UM.Resources import Resources
+from UM.Application import Application
+from UM.Math.Color import Color
+from UM.Preferences import Preferences
+from UM.View.Renderer import Renderer
+
+from UM.View.GL.OpenGL import OpenGL
+
+import math
+
+## Standard view for mesh models. 
```

Comment:
It is different in that it also does the checking if things are inside the build volume and displays overhang. The name "SolidView" comes from the label used it in the view switching dropdown, "Solid". Whether or not that makes sense is up for discussion.